### PR TITLE
Center accordion on larger screens

### LINKED ISSUE #6365: Map content should be centered on full-screen view
The column of content on the /map view should be centered on the page when viewed full screen.

## Mockup
image

I don't think it needs to be centered on the side view, unless centering doesn't interfere with the display.

Ref: https://www.reddit.com/r/FreeCodeCamp/comments/41vr0i/please_bring_back_brownish_fcc_old_version/cz73beo

### PR DESCRIPTION
**Pull request by joshuahenson**
Here is my solution. Let me know or edit it if you feel different sizes for the column would work better.

closes #6365

### CODE REVIEW

**Comment by SaintPeter** [comment]
This does a pretty decent job of centering up the map, but it catches the buttons in the process.

The go from comically narrow:
image

To massively wide:
image

I don't think the button header should be included in the change, just the body of the list.

**Comment by joshuahenson** [comment]
The button sizing was actually done previously by someone else. I hadn't touched them other than to shift the tab spacing in the jade file. 
I removed one of the nested rows on the buttons but while comparing it to the live site I just now noticed that the previous change also added that second button which does nothing. Should  #6358 be rolled back?

**Comment by SaintPeter** [comment]
@joshuahenson The first button should hide completed tasks.

@QuincyLarson Can you take a look at this one.  Is this how the button sizing is supposed to work?  Is the centering action ok?

**Comment by joshuahenson** [comment]
Ah yes, maybe I shouldn't try communicating immediately upon waking up.

Before I got distracted, I was wondering if I should have removed 
`.col-xs-12.col-sm-8.col-sm-offset-2.col-md-6.col-md-offset-3` 
instead of 
`.col-xs-10.col-xs-offset-1.col-sm-6.col-sm-offset-3.col-md-4.col-md-offset-4`

File: server/views/map/show.jade [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-                    .col-xs-12.col-sm-8.col-sm-offset-2.col-md-6.col-md-offset-3
-                      .btn.btn-sm.btn-block.btn-primary.active#manipAll Show incomplete challenges
-                      .btn.btn-sm.btn-block.btn-primary.active#showAll Collapse all
-            hr
+    .text-center.map-fixed-header
+        p Required challenges are marked with a *
+          .row
+            .col-xs-12.col-sm-8.col-sm-offset-2.col-md-6.col-md-offset-3
+              .btn.btn-sm.btn-block.btn-primary.active#manipAll Show incomplete challenges
+              .btn.btn-sm.btn-block.btn-primary.active#showAll Collapse all
```
Comment (SaintPeter) [inline server/views/map/show.jade]: I don't think you should change the button spacing at all.  Or, rather, the buttons are inside a row inside a row and they look "ok" on the current site, I think.  Can you just make the change you made below to the row that contains the accordian?  Or are they nested in some way?

**Comment by joshuahenson** [comment]
I returned the buttons back to the size they are on the live site as they were before the filter button was added.

**Comment by bugron** [comment]
This is on map-aside

**Pull request closed without merging**